test/testmatrix.cpp

```diff
@@ -537,11 +537,40 @@ TEST(Matrix, luDecompPivoting)
     x = A.solve(rhs);
 
     CHECK_EQUAL((4*c - 3*b)/(a*c), x(0));
-    CHECK_EQUAL(3*(b - c)/c, x(1));
+    CHECK_EQUAL(-(3*(-b + c)/c), x(1));
```

Comment:
3*(b - c) is in fact the same as -(3*(-b + c)), except that the former is simpler. i guess this change is needed for the test to pass after the swapEven functionality was introduced, but this seems to be rather a hint to not do manual sign swapping, if the resulting det. is the same, only more complicated?!